src/routers/courseItem.ts

```diff
@@ -37,9 +50,12 @@ cItemRouter.post('/', async (req, res) => {
 
 cItemRouter.put('/', async (req, res) => {
 	try {
-		await create(req.body);
-		console.log('Put Course Item');
-		res.send('Put cItemRouter: ' + req.body.name);
+		const userID = req.header('userID');
+		if (userID) {
+			await update(req.body, userID);
```

Comment:
We should probably build the CourseItem from the req.body in the same way as above (extract to method 😄)